update_maker/cli.py

```diff
@@ -73,7 +90,8 @@ def main():
                 elif command[0] == "help":
                     usage()
                 elif command[0] == "add-pkg":
-                    maker.add_packages(command[1:])
+                    reinstall_not_installed, reinstall_not_removed = reinstall_input(command[1:], maker.not_installed_yet, maker.not_removed_yet)
+                    maker.add_packages(command[1:], reinstall_not_installed, reinstall_not_removed)
```

Comment:
The `UpdaterMaker` already has the lists, no need to sent it to in a method call. Also remove arguments from the method.

Reply:
The `UpdateMaker` doesn't have the reinstall_not_installed and reinstall_not_removed lists, because this list depends on the input of the user. The pkg is only added in those lists if the user input is `y`.

Reply:
Ohhhh, I understand now, never mind then.